traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -154,6 +154,19 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		{1.1, http.MethodGet, `user/{id}/deliveryservices/available/?(\.json)?$`, user.GetAvailableDSes, auth.PrivLevelReadOnly, Authenticated, nil},
 		{1.1, http.MethodPost, `user/login/?$`, login.LoginHandler(d.DB, d.Config), 0, NoAuth, nil},
 
+		//User: CRUD (1.4 then 1.1)
+		//Incrementing version for users because change to Nullable struct. Delete is completely new.
+		{1.4, http.MethodGet, `users/?(\.json)?$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.4, http.MethodGet, `users/{id}?(\.json)?$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.4, http.MethodPut, `users/{id}?(\.json)?$`, api.UpdateHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
```

Comment:
i don't think you need the .json on the PUT and POST

Reply:
also, i am unable to create (POST) a user via TP. here is what i'm sending

![image](https://user-images.githubusercontent.com/251272/46183716-68d98d80-c28f-11e8-9e1c-0546b0579743.png)

and getting a 400 Bad Request with this message

Password must be greater than 7 characters.

Reply:
I'll keep the json since some other endpoints have it. Fixed the bug.